### PR TITLE
Add new licensing API

### PR DESCRIPTION
**Pull request by cvlc**
This API provides easy-access personal licenses for end users.

Also clean up licensing docs and make license stats consistent.

### CODE REVIEW

File: docs/operations/licensing.md [hunk 1]
```diff
@@ -78,55 +78,72 @@ Ondat cluster by browsing the __Licence__ page as shown below:
...
-- **Company** **name** and **role** (optional for Personal licence, mandatory
-  for Commercial licence)
-- **Cluster ID**
-
-In return to the registration, you will receive a licence file with a content
-similar to this one:
-
+## Ondat CLI - Running the CLI
+
+In order to get the CLI running, we must launch it on the target Kubernetes
```
Comment (mdeneva) [inline docs/operations/licensing.md:83]: So, here it is not clear what is meant - who is "WE"? Is it an instruction to the dev? Also the fact the GUI is deleted and CLI added in its place confuses me. If that is an approved change by Rom it is fine by me, but the wording needs to be edited and changed. Everywhere we address the developer/client we write in the imperative form - "You need to do <instruction>"

File: docs/operations/licensing.md [hunk 2]
```diff
@@ -78,55 +78,72 @@ Ondat cluster by browsing the __Licence__ page as shown below:
...
-```
+Be sure to edit the environment variables appropriately for your target cluster, eg. the
+username/password for the administrative user. 
 
-## Ondat GUI - Applying a Licence via the Ondat GUI To apply a licence:
-1.  Browse the __Licence__ page of the Ondat GUI
-1.  Click the __Upgrade__ button, for the specific licence level you purchased
-1.  Paste the licence key into the pop-up window
-1.  To apply the license click __Upgrade__
+Once the pod is launched, we can retrieve it's unique identifier:
```
Comment (mdeneva) [inline docs/operations/licensing.md:128]: Same here - should be "you can retrieve"

File: docs/operations/licensing.md [hunk 3]
```diff
@@ -137,26 +154,91 @@ Created at:       2020-03-25T13:48:33Z (1 hour ago)
...
+- `country` can be a country in ISO 3166 format or the English name of the country
+- `email` will be validated and must be available and non-disposable
+
+> If you receive an `Internal Service Error`, double-check that all the information you've 
+submitted fits the above requirements!
+
+Now check your inbox! You will have received an automatically generated email with a link to
+your license file. If you don't see it, be sure to give it a few minutes and check your spam folder!
+
+The `Download license` button on the email will work as many times as you'd like to retrieve the license -
```
Comment (mdeneva) [inline docs/operations/licensing.md:204]: __Download license__ should be formatted in bold as that is how all screen elements are formatted. This `formatting` is only used for code elements

File: docs/operations/licensing.md (hunk 1 above)
Comment (cvlc) [reply docs/operations/licensing.md]: Thanks, great spot! Corrected.

File: docs/operations/licensing.md (hunk 3 above)
Comment (cvlc) [reply docs/operations/licensing.md]: Fixed! Thanks.

File: docs/operations/licensing.md (hunk 2 above)
Comment (cvlc) [reply docs/operations/licensing.md]: Fixed! Thanks.

**Review by hubvu — COMMENTED** [review]
This looks good to me.

File: docs/operations/licensing.md [hunk 4]
```diff
@@ -78,55 +78,73 @@ Ondat cluster by browsing the __Licence__ page as shown below:
...
 
 ## Ondat CLI - Retrieving a cluster ID via the Ondat CLI
 
-For the purpose of this document, it is assumed that the Ondat CLI has been deployed locally on
-the same machine hosting kubectl. To retrieve the cluster ID using the Ondat
-CLI, perform the following:
+> ⚠️ Make sure that you have the CLI running, as described in the previous step!
+
+```shell
+kubectl -n storageos exec $POD -- storageos get volumes
```
Comment (Arau) [inline docs/operations/licensing.md:144]: This should storageos get licence or get cluster, no?

**Review by Arau — CHANGES_REQUESTED** [review]

File: docs/operations/licensing.md (hunk 4 above)
Comment (cvlc) [reply docs/operations/licensing.md]: Great spot! Updated, also replaced the current output with new-format output from those commands.

**Review by Arau — APPROVED** [review]

**Review by mdeneva — APPROVED** [review]

**Review by mdeneva — APPROVED** [review]

**Pull request merged**